### DISCUSSION TITLE
items are sometimes duplicated in lists

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3685] [Mobile] [answered]
xiaomi redmi note 5 android 9

| 1 | 2 |
| - | - |
| ![1](https://user-images.githubusercontent.com/39104740/113487500-19f4ba80-94c1-11eb-93ca-512884818277.jpg) | ![2](https://user-images.githubusercontent.com/39104740/113487504-1bbe7e00-94c1-11eb-8f1a-cad751c63e49.jpg) |

**Comment by brianlovin** [comment, marked as answer]
@nt4f04uNd do you have a reliable way to reproduce this? e.g. is it something to do with paginating a long list? Or does the view load an initial set of data with duplicates?

**Reply by nt4f04uNd** [reply]
i think this happens when new items are loaded
i don't have a way to reproduce this, this just happens sometimes

**Reply by brianlovin** [reply]
Got it, I'll flag this for our team, but if you find a reliable reproduction flow that would be really helpful 🙏
